### ISSUE TITLE
Save defense placement with city designs

### ISSUE THREAD

**Issue by Deceth** [Deceth/Battle-City#32]
- Save the locations of defensive items when saving city designs
- When loading a city, load defensive items in place.

**Comment by Deceth** [comment]
The downside... when you orb someone, they will be able to attack you again almost instantly. If you've taken damage, you might not have much time to rebuild!

**Comment by Rushing** [comment]
Me thinks, subscriber benefit opertunity here. 8)

**Comment by Deceth** [comment]
I think we definitely need to save defense placement with city designs. However, after an intense battle, both cities often sustain a lot of damage. It's not fair if the other city can rejoin, hit /load and start attacking even before the other city has done rebuilding!

Therefore, ONLY defensive items should be saved. We don't want people to pickup all the bombs, hide them somewhere, save the city, so that when they load it, all their weapons are ready to go.... 

If they actually have to wait for the factories to at least produce the orb and 20 bombs, the other team will have time to rebuild or breathe a little. Remember, they might be under heavy attack from another city as well, so a breather is essential in the tide of battle. 

Also, I would be in favor, if you load a city, to delay the first orb from producing for 3 minutes. This avoids a defeated team using /load to build an orb ASAP and rush in and steal point from another team mixed up in the battle that has survived to this point... 

Basically, if you get orbed, you can't attack again for 3 minutes minimum. Without the ability to load cities it used to take over 10 minutes before you could re-attack, so this isn't too crazy of an idea. You're still in the action again much faster... but at least there is enough delay that others can re-fortify.

> The downside... when you orb someone, they will be able to attack you again almost instantly. If you've taken damage, you might not have much time to rebuild!
